village/src/App.css

```diff
@@ -1,3 +1,25 @@
 .App {
   text-align: center;
+  background-image: url('http://hdqwalls.com/wallpapers/smurfs-the-lost-village-4k-5k-ap.jpg');
+  background-size: 100%;
+  object-fit: cover;
+  background-repeat: no-repeat;
+  height: 100vh;
+  font-family: 'Luckiest Guy', cursive;
+  letter-spacing: 2px;
+  background-attachment: fixed;
+}
+
+.overlay {
+  background: linear-gradient(
+    180deg,
+    rgba(255, 255, 255, 0) 49%,
+    rgba(0, 27, 108, 0.9528186274509804) 100%
+  );
```

Comment:
I love a good linear-gradient done right - great use here, fits well with the overall color scheme.